tide_core.info.yml

```diff
@@ -73,6 +73,7 @@ dependencies:
   - simple_sitemap:simple_sitemap
   - tablefield:tablefield
   - tide_ckeditor:tide_ckeditor
+  - tide_tfa:tide_tfa
```

Comment:
tide_tfa shouldn't be here, which will complicate the dependencies.

Reply:
removed